sky/packages/sky/lib/src/widgets/drag_target.dart

```diff
@@ -222,6 +222,12 @@ class DragRoute extends Route {
     _activeTargetWillAcceptDrop = _activeTarget != null && _activeTarget.didEnter(data);
   }
 
+  void rebuild(BuildContext context) {
+    _entry?.remove();
```

Comment:
this seems like unnecessary thrashing of the list. Is there no way to just update the entry?